docs/pages/choose-an-edition/migrate-to-cloud.mdx

```diff
@@ -323,7 +323,7 @@ your new Teleport cluster, ensure that the setup is complete.
    List all registered Windows desktops:
    
    ```code
-   $ tctl get windows_desktop
+   $ tctl desktops ls

```

Comment:
```suggestion
   $ tctl desktop ls
```

Technically the command is singular. We have an alias so `desktops` will also work, but the help messages will all say `desktop`.